### PR TITLE
handled the priority of ACL rules to be case insensitive

### PR DESCRIPTION
**Pull request by madhanmellanox**
- What I did
I made the priority of ACL rule props in the ACL commands to be case insensitive.

- How I did it
Instead of checking the ACL ruleprops dictionary for "PRIORITY" based key, I made it the actual key to be searched for.

- How to verify it
I verified the fix by testing on my switch using different case sensitive options of keyword "Priority".

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: sonic-utilities-tests/mock_tables/config_db.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "ACL_RULE|DATAACL|RULE_4": {
             "L4_SRC_PORT": "4661",
             "PACKET_ACTION": "FORWARD",
-            "PRIORITY": "9996"
+            "priority": "9996"
```
Comment (stepanblyschak) [inline sonic-utilities-tests/mock_tables/config_db.json:70]: better to create a new rule for the test than modify existing

File: scripts/aclshow [hunk 2]
```diff
@@ -169,8 +169,13 @@ class AclStat(object):
                     self.get_counter_value(rule_key, 'packets') == 'N/A'):
                 continue
             rule = self.acl_rules[rule_key]
+            rule_priority = -1
+            if "PRIORITY" in rule:
+                rule_priority = rule['PRIORITY']
+            elif "priority" in rule:
+                rule_priority = rule['priority']
```
Comment (stepanblyschak) [inline scripts/aclshow:176]: what if priority is 'pRiOrItY' ?

File: sonic-utilities-tests/mock_tables/config_db.json (hunk 1 above)
Comment (madhanmellanox) [reply sonic-utilities-tests/mock_tables/config_db.json]: added a new rule in the next commit.

File: scripts/aclshow (hunk 2 above)
Comment (madhanmellanox) [reply scripts/aclshow]: addressed this code review comment in the next commit.

File: scripts/aclshow [hunk 3]
```diff
@@ -169,8 +169,12 @@ class AclStat(object):
                     self.get_counter_value(rule_key, 'packets') == 'N/A'):
                 continue
             rule = self.acl_rules[rule_key]
+            rule_priority = -1
```
Comment (stepanblyschak) [inline scripts/aclshow:172]: should it be N/A as in acl_loader/main.py?

Comment (madhanmellanox) [reply scripts/aclshow:172]: rule_priority is expecting int, that is why initialized it to -1.

**Pull request merged**